### PR TITLE
[STRATCONN-419] Update namespacing for iOS 14 

### PR DESCRIPTION
**Pull request by briemcnally**
https://paper.dropbox.com/doc/PRD-Analytics-iOS-Namespace-Change--A89lGjxWJiMUgRtRmVL5aN1UAg-FewzU7P1dJM5yKVEgfenA

Segment’s iOS SDK was namespaced to SEGAnalytics under the Objective-C naming convention. As we modernized the SDK to support Swift and more package managers beyond Cocoapods such as Swift Package Manager and Carthage the namespace was renamed to Analytics in-line with Swift idioms as well as Package Manager requirements. It was released as beta. We have received customer feedback that this is leading to namespace collisions and are now going to change this to a much clearer namespace: Segment and the community prefer this as well.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SDK ?= "iphonesimulator"
-DESTINATION ?= "platform=iOS Simulator,name=iPhone X"
+DESTINATION ?= "platform=iOS Simulator,name=iPhone 8"
```
Comment (gpsamson) [inline Makefile:2]: Guessing this has to do with the simulators available on CI?

Comment (briemcnally) [reply Makefile:2]: Yeah also in xcode, iphone X is no longer a supported simulator device

**Review by gpsamson — COMMENTED** [review]
Can we delete the `circle.yml` at the root of the project?

**Comment by briemcnally** [comment]
@gpsamson good catch, sorry forgot to commit that, just pushed

**Review by gpsamson — APPROVED** [review]

**Pull request merged**